scripts/ssl-manager.js

```diff
@@ -635,27 +682,46 @@ function SSLManager(config) {
         return { result : 0 };
     };
 
+    me.attachExtIpIfNeed = function (node) {
+        if (!node.extIPs || node.extIPs.length == 0) {
+            return me.exec.call(nodeManager, nodeManager.attachExtIp, node.id);
+        }
+
+        return { result: 0 };
+    };
+
     me.validateEntryPoint = function validateEntryPoint() {
         var fileName = "validation.sh",
             url = me.getScriptUrl(fileName),
             VALIDATE_IP = "validateExtIP",
-            VALIDATE_DNS = "validateDNSSettings '%(domain)'";
+            VALIDATE_DNS = "validateDNSSettings '%(domain)'",
+            validateNodeId;
 
-        if (!config.withExtIp) VALIDATE_IP = VALIDATE_DNS = 'echo 1';
+        validateNodeId = isBLNodeExists ? nodeManager.getBalancerMasterNode().id : config.nodeId;
```

Comment:
it is better to you getNode or getEntryPointNode here

Reply:
tested. A _getBalancerMasterNode_ already has a bl node as far as **getEntryPointGroup** executed before